Slim/Plugin/FullTextSearch/Plugin.pm

```diff
@@ -560,6 +565,10 @@ sub _rebuildIndex {
 	$scanlog->error("Create fulltext index for works");
 	$progress && $progress->update(string('WORKS'));
 	Slim::Schema->forceCommit if main::SCANNER;
+	$sql = SQL_DROP_WORKTEMP;
+	$dbh->do($sql) or $scanlog->error($dbh->errstr);
+	$sql = SQL_CREATE_WORKTEMP;
+	$dbh->do($sql) or $scanlog->error($dbh->errstr);
```

Comment:
There's non reason really to assign a constant to the variable here. Just execute the constant?

Reply:
done